Add PC Fame script commands (#7310)
rathena/rathena: doc/script_commands.txt

- Excludes ACHIEVEINFO_LEVEL and ACHIEVEINFO_SCORE.

---------------------------------------

*addfame(<amount>,{,<char id>})

> Increase the fame of character by a specific amount.
> Note: Only work for blacksmith, alchemist and taekwon class.

Reviewer: ```suggestion
Increases the fame of the attached player or the supplied <char id> by the <amount> given.
Note: Only works with Blacksmith, Alchemist and Taekwon classes.
```

---------------------------------------

*getfame({<char id>})

> Gets the fame for the attached player or the given character ID.

Reviewer: ```suggestion
Gets the fame points of the attached player or the supplied <char id>.
Note: Only works with Blacksmith, Alchemist and Taekwon classes.
```

---------------------------------------

*getfamerank({<char id>})

> Check if player is in the fame rankers list of its job.

Reviewer: ```suggestion
Gets the fame rank of the attached player or the supplied <char id>.
```

> Returns fame rank (start from 1 to MAX_FAME_LIST), else 0.

Reviewer: ```suggestion
Returns fame rank (start from 1 to MAX_FAME_LIST), else 0.
Note: Only works with Blacksmith, Alchemist and Taekwon classes.
```

- Excludes ACHIEVEINFO_LEVEL and ACHIEVEINFO_SCORE.

---------------------------------------

*addfame(<amount>,{,<char id>})

Increases the fame of the attached player or the supplied <char id> by the <amount> given.

> Note: Only works with Blacksmith, Alchemist and Taekwon classes.

Reviewer 2: ```suggestion
Note: Only works with classes that use the ranking system.
```
Would be good to generalize it, so that we do not have to think about updating all references, when a rank is added to a class.

---------------------------------------

*getfame({<char id>})

Gets the fame points of the attached player or the supplied <char id>.

> Note: Only works with Blacksmith, Alchemist and Taekwon classes.

Reviewer 2: ```suggestion
```
Not checked source side.

---------------------------------------

*getfamerank({<char id>})

Returns fame rank (start from 1 to MAX_FAME_LIST), else 0.

> Note: Only works with Blacksmith, Alchemist and Taekwon classes.

Reviewer 2: ```suggestion
```
Not checked source side.
